File: java/URI_1009.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1009 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double fixsalary, totvendas, salario;
		
		String name = sc.next();
		fixsalary = sc.nextDouble();
		totvendas = sc.nextDouble();
		
		salario = fixsalary + totvendas * 0.15;
		
		System.out.printf("TOTAL = R$ %.2f%n", salario);
		
		
		sc.close();

	}

}

File: kotlin/URI_1009.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val nome = sc.next()
    val salarioFixo = sc.nextDouble()
    val vendas = sc.nextDouble()
    val salarioTotal = salarioFixo + vendas * 0.15
    println("TOTAL = R$ ${"%.2f".format(salarioTotal)}")
}